samples/built-in-policy/require-storageaccount-encryption/azurepolicy.json

```diff
@@ -1,28 +0,0 @@
-{
-  "properties": {
-    "displayName": "Require storage account encryption",
```

Comment:
>"Require storage account encryptio [](start = 19, length = 34)

this can't be removed as long as the sample still exists in the docs page ([https://docs.microsoft.com/en-us/azure/governance/policy/samples/require-storage-account-encryption](https://docs.microsoft.com/en-us/azure/governance/policy/samples/require-storage-account-encryption))

Reply:
I will wait for David's response.